IPythEvents.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+interface IPythEvents {
```

Comment:
nit: could we add a comment explaining the purpose of this interface?

Reply:
Done.